diff --git a/layout/base/AccessibleCaret.h b/layout/base/AccessibleCaret.h
index 9b0c33a1df1ad717942832aff89e935dd9a236f9..611abcff6bc09171cda1f02a7caf58b4ff158aac 100644
--- a/layout/base/AccessibleCaret.h
+++ b/layout/base/AccessibleCaret.h
@@ -80,6 +80,8 @@ class AccessibleCaret {
 
   virtual void SetAppearance(Appearance aAppearance);
 
+  void SetVisuallyHiddenForNativeSelection(bool aHidden);
+
   // Return true if current appearance is either Normal, NormalNotShown, Left,
   // or Right.
   bool IsLogicallyVisible() const { return mAppearance != Appearance::None; }
