diff --git a/netwerk/protocol/http/nsHttpChannel.cpp b/netwerk/protocol/http/nsHttpChannel.cpp
index 42ee24d0b93641cae705d8709467141125a3f7ea..ae4da29e02aa893185d1790ff4de6495b809205d 100644
--- a/netwerk/protocol/http/nsHttpChannel.cpp
+++ b/netwerk/protocol/http/nsHttpChannel.cpp
@@ -1240,9 +1240,10 @@ nsresult nsHttpChannel::ContinueOnBeforeConnect(bool aShouldUpgrade,
     NS_ENSURE_SUCCESS(rv, rv);
 
     wtconSettings->GetServerCertificateHashes(aServerCertHashes);
-    gHttpHandler->ConnMgr()->StoreServerCertHashes(
+    DebugOnly<nsresult> storeRv = gHttpHandler->StoreServerCertHashes(
         mConnectionInfo, gHttpHandler->IsHttp2Excluded(mConnectionInfo),
         !Http3Allowed(), std::move(aServerCertHashes));
+    MOZ_ASSERT(NS_SUCCEEDED(storeRv));
   }
 
   if (ShouldIntercept()) {
